pkg/security/probe/perf_buffer_monitor.go

```diff
@@ -349,14 +349,12 @@ func (pbm *PerfBufferMonitor) CountEvent(eventType model.EventType, timestamp ui
 func (pbm *PerfBufferMonitor) sendEventsAndBytesReadStats(client statsd.ClientInterface) error {
 	var count int64
 	var err error
-	tags := []string{pbm.probe.config.StatsTagsCardinality, "", ""}
 
 	for m := range pbm.stats {
-		tags[1] = fmt.Sprintf("map:%s", m)
 		for cpu := range pbm.stats[m] {
 			for eventType := range pbm.stats[m][cpu] {
 				evtType := model.EventType(eventType)
-				tags[2] = fmt.Sprintf("event_type:%s", evtType)
+				tags := pbm.buildTags(fmt.Sprintf("map:%s", m), fmt.Sprintf("event_type:%s", evtType))
```

Comment:
can we extract the map tag to the top of the loop ? I believe we will call the Sprintf for each cpu/event type here ?

Reply:
We can't because of the event type :(

Reply:
I'm talking about only `fmt.Sprintf("map:%s", m) `